pom.xml

```diff
@@ -16,12 +16,13 @@
                 this result is useless to us -->
         <findbugs.failOnError>false</findbugs.failOnError>
 
-		<selenium.short.version>2.53</selenium.short.version>
-		<selenium.version>${selenium.short.version}.0</selenium.version>
-        <htmlunit.version>2.20</htmlunit.version>
+		<selenium.short.version>3.0</selenium.short.version>
+		<selenium.version>${selenium.short.version}.1</selenium.version>
+        <htmlunit.version>2.21</htmlunit.version>
+		<java.level>8</java.level>
 	</properties>
 	<artifactId>selenium</artifactId>
-	<version>2.53.2-SNAPSHOT</version>
+	<version>3.0.0-SNAPSHOT</version>
```

Comment:
By the way, you seem to have updated the plugin version manually in the pom file. You should not do this. Jenkins has a maven build-release plugin workflow where upon release the version number for the next release, and next snapshot version are provided. Please revert that part of your pull request.